src/e2e-test/features/salesforcesink/RunTime.feature

```diff
@@ -126,4 +126,42 @@ Feature: Salesforce Sink - Run time Scenarios
     Then Wait till pipeline is in running state
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
-    Then Close the pipeline logs
\ No newline at end of file
+    Then Close the pipeline logs
+
+  @SINK-TS-SF-RNTM-05 @BQ_SOURCE_TEST @DELETE_OBJECT
+  Scenario: To verify data is getting transferred from Big Query source to Salesforce sink successfully
```

Comment:
Same comment update the above tests and remove this one.

Reply:
done